src/elements/content-sidebar/DetailsSidebar.js

```diff
@@ -329,7 +329,12 @@ class DetailsSidebar extends React.PureComponent<Props, State> {
 
         // TODO: Add loading indicator and handle errors once file call is split out
         return (
-            <SidebarContent className="bcs-details" title={SidebarUtils.getTitleForView(SIDEBAR_VIEW_DETAILS)}>
+            <SidebarContent
+                className="bcs-details"
+                title={SidebarUtils.getTitleForView(SIDEBAR_VIEW_DETAILS)}
+                aria-labelledby="sidebardetails"
+                id="sidebardetails-content"
```

Comment:
We technically support having multiple of the same types of Element on the page at one time. I think we'll need to add some form of unique id generation to the mix here.